lib/element-commands.js

```diff
@@ -316,6 +320,12 @@ _.each(utils.elementFuncTypes, function(type) {
     elementCommands.element.apply(this, [utils.elFuncFullType(type), value, cb]);
   };
 
+  elementCommands['waitForElement' + utils.elFuncSuffix(type)] = function() {
```

Comment:
I'm confused: I thought we already had these?

Reply:
wd's internals requires two distinct instances of functions. One for "browser" and another for "element". And they could be implemented a bit differently. wd has only browser instance. Check this: https://github.com/admc/wd/issues/365